pkgs/by-name/pi/piliplus/package.nix

```diff
@@ -2,103 +2,98 @@
   lib,
   fetchFromGitHub,
   flutter335,
-  makeShellWrapper,
   makeDesktopItem,
   copyDesktopItems,
   alsa-lib,
   mpv-unwrapped,
   libplacebo,
+  libappindicator,
 }:
 
 let
-  version = "1.1.4.4";
-  rev = "f0f52246777f2640025048f561e908cf1d3c3ead";
+  version = "1.1.4.8";
+  rev = "6ef9a24ed109d251eee4b0e4d6ed10932d90ef9d";
+  revCount = "4176"; # this line is matched by regexp in update.rb
+  commitDate = "1759647826"; # this line is matched by regexp in update.rb
```

Comment:
I would highly recommend replacing those values with placeholders to simplify the update script greately.